rules/framework.bzl

```diff
@@ -699,70 +726,130 @@ def _bundle_dynamic_framework(ctx, is_extension_safe, avoid_deps):
         attr = ctx.attr,
         res_attrs = ["infoplists"],
     )
-    link_result = linking_support.register_binary_linking_action(
-        ctx,
-        avoid_deps = avoid_deps,
-        entitlements = None,
-        extra_linkopts = extra_linkopts,
-        platform_prerequisites = platform_prerequisites,
-        stamp = ctx.attr.stamp,
-    )
+    if apple_api_version == "3.0":
```

Comment:
For future readers it might get confusing to understand what `apple_api_version` means, should we explicitly give it the rule set's name e.g. `rules_apple_api_version`?

Reply:
SGTM

Reply:
Updated